devices/gpu/nvidia/device.go

```diff
@@ -37,10 +37,10 @@ const (
 var (
 	// pluginInfo describes the plugin
 	pluginInfo = &base.PluginInfoResponse{
-		Type:             base.PluginTypeDevice,
-		PluginApiVersion: "0.0.1", // XXX This should be an array and should be consts
-		PluginVersion:    "0.1.0",
-		Name:             pluginName,
+		Type:              base.PluginTypeDevice,
+		PluginApiVersions: []string{"v0.2.0"},
```

Comment:
should this be a const?

Reply:
Ha good catch! It was that way as I was manually testing to check that API version validation works!